infra/gcp/infra.yaml

```diff
@@ -334,5 +334,6 @@ infra:
       k8s-staging-test-infra:
       k8s-staging-tg-exporter:
       k8s-staging-gmsa-webhook:
+      k8s-staging-op-readiness:
```

Comment:
I know were pushing up against a character limit here but should we try and fit windows in the name?
maybe `k8s-staging-win-op-rdnss`?